docs/privacy/policy.md

```
@@ -1,67 +1,77 @@
+        - 📝 Certificate requests are purged after 24h
+        - 🔑 Private keys generated client-side
+        - 📜 CSR generated by the CLI tool contains only IP address
 
-- **Push Notification Handling**: 
-    - Push notifications are routed through the cloud server.
-    - These notifications contain only the device's FCM token.
-    - No SMS content or recipient information is included in these notifications.
+---
 
-### Certificate Authority (CA)
+## User Privacy :material-shield-account:
```

markdownlint-cli2 (0.17.2)

Trailing punctuation in heading
Punctuation: ':'

(MD026, no-trailing-punctuation)